virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -209,6 +209,13 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 * The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://issues.redhat.com/browse/CNV-33834[*CNV-33834*])
 ** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].
 
+[discrete]
+[id="virt-4-15-ki-networking"]
+==== Networking
+//CNV-38593: RN: Known issue Remove invalid annotation from "OVN Kubernetes secondary localnet network"
+* The invalid annotation `k8s.v1.cni.cncf.io/resourceName: openshift.io/` appears during the creation of a Network Access Device (NAD) with the `OVN Kubernetes secondary localnet network` network type. This annotation prevents the starting of the VM.
```

Comment:
I am wondering if backticks should be applied to "OVN Kubernetes secondary localnet network" or not - Is this network type a part of a web console? In that case, backticks might not go here.

Reply:
Fixed